charts/gardener-extension-networking-cilium/templates/rbac.yaml

```diff
@@ -87,6 +87,18 @@ rules:
   - networkpolicies
   verbs:
   - "*"
+- apiGroups:
+  - policy
+  resources:
+  - poddisruptionbudgets
+  verbs:
+  - create
+  - get
+  - list
+  - watch
+  - patch
+  - update
+  - delete
```

Comment:
Why do we need this? If it is required, why do we not have this for the admission controller?

Reply:
You're right, it's not needed since the `PDB` is deployed via  a `ManagedResource`.